src/argbash.m4

```diff
@@ -77,55 +72,56 @@ interpret_error()
 # $2: The original intended output file
 define_file_metadata()
 {
-	local _defines='' _intended_destination="$ARGBASH_INTENDED_DESTINATION" _input_dirname _output_dirname
-	test -n "$_intended_destination" || _intended_destination="$2"
+	ARGBASH_INTENDED_DESTINATION="${ARGBASH_INTENDED_DESTINATION=}"
+	SPURIONS_OUTPUT_SUFFIX="${SPURIONS_OUTPUT_SUFFIX=}"
```

Comment:
I think that by `SPURIONS` I actually meant `SPURIOUS`, as it is used with `basename` that removes extensions.
Could you please fix that one, as you have fixed plenty of issues already?

Reply:
Sure, not a problem.